crates/services/upgradable-executor/src/executor.rs

```diff
@@ -604,6 +607,44 @@ where
         }
     }
 
+    /// Wasm support is disabled in the current build.
+    #[cfg(not(feature = "wasm-executor"))]
+    pub fn uploaded_wasm_is_valid(&self, _wasm_root: &Bytes32) -> ExecutorResult<()> {
+        Err(ExecutorError::NoWasmSupport)
+    }
```

Comment:
I think we don't need to modify the `ExecutorError` and instead create an `UpgradableExecutorError` since `fuel-core-executor` doesn't know about WASM

Reply:
Done in 71744b63c.